Data/ByteString/Builder/RealFloat.hs

```diff
@@ -72,12 +82,18 @@ module Data.ByteString.Builder.RealFloat
 
 import Data.ByteString.Builder.Internal (Builder)
 import qualified Data.ByteString.Builder.RealFloat.Internal as R
+import Data.ByteString.Builder.RealFloat.Internal (FloatFormat(..), fScientific, fGeneric)
+import Data.ByteString.Builder.RealFloat.Internal (positiveZero, negativeZero)
```

Comment:
```suggestion
import Data.ByteString.Builder.RealFloat.Internal (FloatFormat(..), fScientific, fGeneric, SpecialStrings(..))
```

Importing `positiveZero` and `negativeZero` directly by name doesn't work so well due to `NoFieldSelectors`/`DuplicateRecordFields`